docs/data-collection/scanning-notes.md

```diff
@@ -36,27 +36,28 @@
 
     ![adjustFOV.jpg](../assets/images/adjustFOV.jpg)
 
-## Eye-tracker calibration
-- [ ] Inform the participant about the calibration process (the [scanning protocol](scanning.md) provides you with scripts)
-- [ ] Run the experiment by pressing the green play button. 
-- [ ] Enter the session and participant number in the pop up window. The Eyelink system setup page opens.
-- [ ] Press <span class="keypress">C</span> to open the calibration mode to perform the calibration and validation.
-- [ ] Exit the calibration mode by clicking on  <span class="keypress">ESC</span> .
-- [ ] Once the calibration of the ET is concluded, hit the <span class="keypress">Esc</span> key on the laptop *{{ secrets.hosts.psychopy | default("███") }}* to exit the calibration mode continue the task's program.
-
-
-- [ ] Launch the ET calibration by pressing <span class="keypress">C</span> on the laptop keyboard or by clicking on Calibration on the ET interface:
-    - [ ] When the gaze is stable, the button `Accept fixation` appears green, then you can manually click on it to validate the first position.
+## Eye-Tracker Calibration Process
+
+- [ ] Brief the participant about the calibration process. Refer to the [scanning protocol](scanning.md) for provided scripts.
+- [ ] Start the experiment by pressing the green play button.
+- [ ] On the eye-tracking (ET) computer, ensure that the appropriate calibration type is selected (9 points for PCT and 5 points for resting state and breath holding tasks):
+    - [ ] Click on `Set Options` located on the right side of the screen.
+    - [ ] Under `Calibration type` in the top left corner, choose the image containing either 9 or 5 points.
+      
+![9-points_calibration](../assets/images/9-points_calibration.jpg)
+      
+- [ ] Two crosses should appear on the ET computer screen: one at the center of the pupil and the other at the center of the corneal reflection. If they're not visible, proceed sequentially with the following steps:
+    - [ ] Adjust the focus of the ET camera and the pupil detection threshold. If the cross remains invisible,
+    - [ ] Realign the mirror frame position by sliding it along the rails attached to the coil. If the cross still isn't visible,
+    - [ ] Re-adjust the participant's head positioning inside the coil. If the cross is still not visible,
+    - [ ] Make slight adjustments to the mirror's vertical position (be cautious as mentioned earlier). Even a small adjustment can impact calibration and eye positioning. If the cross remains invisible,
```

Comment:
```suggestion
    - [ ] **If the cross remains invisible**, revise the vertical position of the infrared mirror with respect to the standard mirror (this will require new gloves and replacing the tape to stick the infrared mirror).
        Even a tiny adjustment can impact calibration and eye positioning. 
```